### PR TITLE
sys: Correct types of sys.displayhook, sys.excepthook and variants

### PR DESCRIPTION
**Pull request by janverb**
`displayhook` should accept any object, not just integers.

`displayhook` and `excepthook` may be assigned to, so they should be `Callable` values rather than function definitions. That way it's fine to assign a function with different argument names. Their dunder variants are supposed to be constant.

### CODE REVIEW

**Comment by srittau** [comment]
Sounds reasonable. PR welcome!

**Comment by janverb** [comment]
This is a PR :slightly_smiling_face:

**Comment by srittau** [comment]
Whoops, the new notification UI is confusing sometimes ...

File: stdlib/3/sys.pyi [hunk 1]
```diff
@@ -28,8 +28,8 @@ builtin_module_names: Sequence[str]  # actually a tuple of strings
 copyright: str
 # dllhandle = 0  # Windows only
 dont_write_bytecode: bool
-__displayhook__: Any  # contains the original value of displayhook
-__excepthook__: Any  # contains the original value of excepthook
+displayhook: Callable[[object], None]
+excepthook: Callable[[Type[BaseException], BaseException, TracebackType], None]
```
Comment (srittau) [inline stdlib/3/sys.pyi]: Since the return value of these functions is (most likely, I did not check) ignored, the return type should be `Any`.

**Pull request merged**